langium/er-dsl/src/language/PlantUmlGeneration/generate-plantuml.ts

```diff
@@ -78,19 +66,6 @@ function getDataTypeString(attribute: Attribute): string {
     return `${attribute.datatype?.name}`;
 }
```

Comment:
If these are put in a seperate file besides this one, then they can be safely imported into the classes.
That will provide a much cleaner function signature, imo.
_If_ it actually serves a purpose, and the idea is that the same method in the class can be provided with different external functions, then i dont mind the implementation. If it is purely done to solve the problem of circular imports, then i prefer to change it.

(Imagine a file besides `generate-plantuml.ts` possibly `plantuml-utils.ts`)

Reply:
I cant add additional comments, but the commented lines in the function body should be deleted.
If they serve an important purpose add a comment explaining why they remain.